src/components/Tasks/TaskList.tsx

```diff
@@ -31,9 +38,26 @@ interface PropsType {
 
 export default React.memo(function TaskList(props: PropsType) {
   const [showCompleted, setShowCompleted] = React.useState(false);
-  const entries = props.entries.filter((x) => showCompleted || !x.finished);
+  const settings = useSelector((state: StoreState) => state.settings.taskSettings);
+  const { filterBy } = settings;
+  const theme = useTheme();
+
+  const potentialEntries = props.entries.filter((x) => showCompleted || !x.finished);
+  let entries;
+  if (filterBy?.startsWith('tag')) {
+    const tag = filterBy.slice(4);
+    entries = potentialEntries.filter((x) => x.tags.includes(tag));
+  } else {
+    entries = [...potentialEntries];
```

Comment:
Why are you cloning the list? Just use the potentialEntries list directly. Cloning for no reason is inefficient. :)

Reply:
This was actually intentional. There was some weirdness with `entries` being a reference to `potentialEntries`. Since I was sorting `entries`, it was also sorting `potentialEntries`. And since the tags list used `potentialEntries`, the order of the tags would unexpectedly change when selecting different filtering options.

I found a dumb simple solution to this though: just sort the entries _after_ getting the tags.

Reply:
You should probably filter the tags alphabetically anyway, so I guess it doesn't matter if you sort before or after.